src/utils.coffee

```diff
@@ -204,3 +204,134 @@ exports.getOSVersion = (path) ->
 	.catch (err) ->
 		console.log('Could not get OS Version: ', err, err.stack)
 		return undefined
+
+exports.defaultVolumes = {
+	'/data': {}
+	'/lib/modules': {}
+	'/lib/firmware': {}
+	'/host/var/lib/connman': {}
+	'/host/run/dbus': {}
+}
+
+exports.defaultBinds = (dataPath) ->
+	return [
+		config.dataPath + '/' + dataPath + ':/data'
+		'/lib/modules:/lib/modules'
+		'/lib/firmware:/lib/firmware'
+		'/run/dbus:/host_run/dbus'
+		'/run/dbus:/host/run/dbus'
+		'/etc/resolv.conf:/etc/resolv.conf:rw'
+		'/var/lib/connman:/host/var/lib/connman'
+	]
+
+exports.validComposeOptions = [
+	'command'
+	'entrypoint'
+	'env_file'
+	'environment'
+	'expose'
+	'image'
+	'labels'
+	'ports'
+	'stop_signal'
+	'volumes'
+	'user'
+	'working_dir'
+	'cap_add'
+	'cap_drop'
+	'devices'
+	'dns'
+	'dns_search'
+	'tmpfs'
+	'extra_hosts'
+	'links'
+	'net'
+	'network_mode'
+	'ulimits'
+	'volumes_from'
+	'cpu_shares'
+	'cpu_quota'
+	'cpuset'
+	'domainname'
+	'hostname'
+	'mac_address'
+	'mem_limit'
+	'memswap_limit'
+	'privileged'
+	'tty'
+	'read_only'
+	'shm_size'
+	'ipc'
+	'restart'
+	'security_opt'
+	'networks'
+	'pid'
+]
+
+exports.validContainerOptions = [
+	'Hostname'
+	'Domainname'
+	'User'
+	'Tty'
+	'Env'
+	'Labels'
+	'Cmd'
+	'Entrypoint'
+	'Image'
+	'Volumes'
+	'WorkingDir'
+	'NetworkDisabled'
+	'ExposedPorts'
+	'HostConfig'
+]
```

Comment:
Need 'Name' also to be in this list. We use Name to name our containers in the create API